tests/tests_plotting/edgecases/test_plot_pytrendy_edgecases.py

```
@@ -68,10 +69,14 @@ def test_plot_debug_add_vertical_lines(self):
         # ------ pt.detect_trends() [part 1]
         # unwrapped-equivalent to disable grouping at a lower level     
-        df[date_col] = pd.to_datetime(df[date_col])
+        external_index = pd.to_datetime(df[date_col])
+        internal_index = np.arange(len(df))
+        index_lookup = {internal_index[i] : external_index[i] for i in range(len(internal_index))}
+    
+        df[date_col] = internal_index
         df.set_index(date_col, inplace=True)
         df = df[[value_col]]
-        method_params = {'abrupt_padding':0, 'avoid_noise':True}
+        method_params = dict(is_abrupt_padded=False, abrupt_padding=28, avoid_noise=True)
```

Ruff (0.15.20)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)